TestCases/serial_regression.py

```diff
@@ -363,6 +363,17 @@ def main():
     turb_naca0012_sst_sust.tol       = 0.00001
     test_list.append(turb_naca0012_sst_sust)
 
+    # NACA0012 (SST, fixed values for turbulence quantities)
+    turb_naca0012_sst_fixedvalues           = TestCase('turb_naca0012_sst_fixedvalues')
+    turb_naca0012_sst_fixedvalues.cfg_dir   = "rans/naca0012"
+    turb_naca0012_sst_fixedvalues.cfg_file  = "turb_NACA0012_sst_fixedvalues.cfg"
+    turb_naca0012_sst_fixedvalues.test_iter = 10
+    turb_naca0012_sst_fixedvalues.test_vals = [-9.562435, -1.566603, 1.022029, 0.040549]
```

Comment:
just a note: One can add as many screen outputs as one wants here. I think it is a good thing to add some (even arbitrary) screen outputs in order to slightly increase the coverage. It might save the e.g. LINSOL_RESIDUAL screen output in that solver one day.

Reply:
You mean to add `LINSOL_RESIDUAL` to the line

    SCREEN_OUTPUT= (INNER_ITER, RMS_DENSITY, RMS_TKE, RMS_DISSIPATION, LIFT, DRAG)

I'd then just update the new testcase, or is it better to cover it in all of the three testcases of this family (pure SST, SST_SUST, and fixedvalues)?

Reply:
It is a more a general thing (for future PR's) because in this case you could also just add the 5th line which would be RMS_DENSITY which would already help increasing test coverage (and maybe even INNER_ITER as 6th one). You can add as many 'test.vals' as you like and should maybe use that to safeguard some things which might otherwise go untested. So if you had
``` 
SCREEN_OUTPUT= (INNER_ITER, RMS_DENSITY, RMS_TKE, RMS_DISSIPATION, LIFT, DRAG, LINSOL_RESIDUAL)
```
you could 
```python
turb_naca0012_sst_fixedvalues.test_vals = [9, -3.456789, -9.562435, -1.566603, 1.022029, 0.040549, 0.012345]
```
with the first 2 and the last value made up.

Reply:
You need to leave inner iter out of the test_vals, but include it in the config.
+1 vote to all your comments.